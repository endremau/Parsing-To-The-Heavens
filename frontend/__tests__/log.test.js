import React from 'react';
import renderer from 'react-test-renderer'
import Log from '../src/components/log'

it('renders correctly', ()=>{
    const tree=renderer
        .create(<Log/>)
        .toJSON();
    expect(tree).toMatchSnapshot()
})