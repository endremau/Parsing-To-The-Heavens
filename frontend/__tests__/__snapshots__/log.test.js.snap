// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "marginLeft": "20vw",
      "paddingTop": "50px",
    }
  }
>
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <div
        className="ui left icon input"
      >
        <input
          name="formDesc"
          onChange={[Function]}
          placeholder="Description"
          type="text"
          value=""
        />
        <i
          aria-hidden="true"
          className="search icon"
          onClick={[Function]}
        />
      </div>
    </div>
    <div
      className="field"
    >
      <div
        aria-expanded={false}
        aria-multiselectable={true}
        className="ui fluid multiple selection dropdown"
        name="formDevIds"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseDown={[Function]}
        role="listbox"
        tabIndex={0}
      >
        <div
          aria-live="polite"
          className="default text"
          role="alert"
        >
          Device ids
        </div>
        <i
          aria-hidden="true"
          className="dropdown icon"
          onClick={[Function]}
        />
        <div
          className="menu transition"
        >
          <div
            aria-checked={false}
            aria-selected={true}
            className="selected item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              1
            </span>
          </div>
          <div
            aria-checked={false}
            aria-selected={false}
            className="item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              2
            </span>
          </div>
          <div
            aria-checked={false}
            aria-selected={false}
            className="item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              3
            </span>
          </div>
          <div
            aria-checked={false}
            aria-selected={false}
            className="item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              4
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="field"
    >
      <div
        aria-expanded={false}
        aria-multiselectable={true}
        className="ui fluid multiple selection dropdown"
        name="formAdaptIds"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseDown={[Function]}
        role="listbox"
        tabIndex={0}
      >
        <div
          aria-live="polite"
          className="default text"
          role="alert"
        >
          Adaption ids
        </div>
        <i
          aria-hidden="true"
          className="dropdown icon"
          onClick={[Function]}
        />
        <div
          className="menu transition"
        >
          <div
            aria-checked={false}
            aria-selected={true}
            className="selected item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              1
            </span>
          </div>
          <div
            aria-checked={false}
            aria-selected={false}
            className="item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              2
            </span>
          </div>
          <div
            aria-checked={false}
            aria-selected={false}
            className="item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              3
            </span>
          </div>
          <div
            aria-checked={false}
            aria-selected={false}
            className="item"
            onClick={[Function]}
            role="option"
            style={
              Object {
                "pointerEvents": "all",
              }
            }
          >
            <span
              className="text"
            >
              4
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="field"
    >
      <span
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        Date
        <i
          aria-hidden="true"
          className="arrows alternate horizontal link icon"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          style={
            Object {
              "marginLeft": "5px",
            }
          }
        />
      </span>
    </div>
    <div
      className="one fields"
    >
      <div
        className="sixteen wide field"
      >
        <div
          className="rdt"
        >
          <input
            className="form-control"
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            type="text"
            value=""
          />
          <div
            className="rdtPicker"
          >
            <div
              className="rdtDays"
            >
              <table>
                <thead>
                  <tr>
                    <th
                      className="rdtPrev"
                      onClick={[Function]}
                    >
                      <span>
                        ‹
                      </span>
                    </th>
                    <th
                      className="rdtSwitch"
                      colSpan={5}
                      data-value={3}
                      onClick={[Function]}
                    >
                      april 2019
                    </th>
                    <th
                      className="rdtNext"
                      onClick={[Function]}
                    >
                      <span>
                        ›
                      </span>
                    </th>
                  </tr>
                  <tr>
                    <th
                      className="dow"
                    >
                      ma
                    </th>
                    <th
                      className="dow"
                    >
                      ti
                    </th>
                    <th
                      className="dow"
                    >
                      on
                    </th>
                    <th
                      className="dow"
                    >
                      to
                    </th>
                    <th
                      className="dow"
                    >
                      fr
                    </th>
                    <th
                      className="dow"
                    >
                      lø
                    </th>
                    <th
                      className="dow"
                    >
                      sø
                    </th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td
                      className="rdtDay rdtOld"
                      data-value={25}
                      onClick={[Function]}
                    >
                      25
                    </td>
                    <td
                      className="rdtDay rdtOld"
                      data-value={26}
                      onClick={[Function]}
                    >
                      26
                    </td>
                    <td
                      className="rdtDay rdtOld"
                      data-value={27}
                      onClick={[Function]}
                    >
                      27
                    </td>
                    <td
                      className="rdtDay rdtOld"
                      data-value={28}
                      onClick={[Function]}
                    >
                      28
                    </td>
                    <td
                      className="rdtDay rdtOld"
                      data-value={29}
                      onClick={[Function]}
                    >
                      29
                    </td>
                    <td
                      className="rdtDay rdtOld"
                      data-value={30}
                      onClick={[Function]}
                    >
                      30
                    </td>
                    <td
                      className="rdtDay rdtOld"
                      data-value={31}
                      onClick={[Function]}
                    >
                      31
                    </td>
                  </tr>
                  <tr>
                    <td
                      className="rdtDay"
                      data-value={1}
                      onClick={[Function]}
                    >
                      1
                    </td>
                    <td
                      className="rdtDay"
                      data-value={2}
                      onClick={[Function]}
                    >
                      2
                    </td>
                    <td
                      className="rdtDay"
                      data-value={3}
                      onClick={[Function]}
                    >
                      3
                    </td>
                    <td
                      className="rdtDay"
                      data-value={4}
                      onClick={[Function]}
                    >
                      4
                    </td>
                    <td
                      className="rdtDay"
                      data-value={5}
                      onClick={[Function]}
                    >
                      5
                    </td>
                    <td
                      className="rdtDay"
                      data-value={6}
                      onClick={[Function]}
                    >
                      6
                    </td>
                    <td
                      className="rdtDay"
                      data-value={7}
                      onClick={[Function]}
                    >
                      7
                    </td>
                  </tr>
                  <tr>
                    <td
                      className="rdtDay"
                      data-value={8}
                      onClick={[Function]}
                    >
                      8
                    </td>
                    <td
                      className="rdtDay rdtToday"
                      data-value={9}
                      onClick={[Function]}
                    >
                      9
                    </td>
                    <td
                      className="rdtDay"
                      data-value={10}
                      onClick={[Function]}
                    >
                      10
                    </td>
                    <td
                      className="rdtDay"
                      data-value={11}
                      onClick={[Function]}
                    >
                      11
                    </td>
                    <td
                      className="rdtDay"
                      data-value={12}
                      onClick={[Function]}
                    >
                      12
                    </td>
                    <td
                      className="rdtDay"
                      data-value={13}
                      onClick={[Function]}
                    >
                      13
                    </td>
                    <td
                      className="rdtDay"
                      data-value={14}
                      onClick={[Function]}
                    >
                      14
                    </td>
                  </tr>
                  <tr>
                    <td
                      className="rdtDay"
                      data-value={15}
                      onClick={[Function]}
                    >
                      15
                    </td>
                    <td
                      className="rdtDay"
                      data-value={16}
                      onClick={[Function]}
                    >
                      16
                    </td>
                    <td
                      className="rdtDay"
                      data-value={17}
                      onClick={[Function]}
                    >
                      17
                    </td>
                    <td
                      className="rdtDay"
                      data-value={18}
                      onClick={[Function]}
                    >
                      18
                    </td>
                    <td
                      className="rdtDay"
                      data-value={19}
                      onClick={[Function]}
                    >
                      19
                    </td>
                    <td
                      className="rdtDay"
                      data-value={20}
                      onClick={[Function]}
                    >
                      20
                    </td>
                    <td
                      className="rdtDay"
                      data-value={21}
                      onClick={[Function]}
                    >
                      21
                    </td>
                  </tr>
                  <tr>
                    <td
                      className="rdtDay"
                      data-value={22}
                      onClick={[Function]}
                    >
                      22
                    </td>
                    <td
                      className="rdtDay"
                      data-value={23}
                      onClick={[Function]}
                    >
                      23
                    </td>
                    <td
                      className="rdtDay"
                      data-value={24}
                      onClick={[Function]}
                    >
                      24
                    </td>
                    <td
                      className="rdtDay"
                      data-value={25}
                      onClick={[Function]}
                    >
                      25
                    </td>
                    <td
                      className="rdtDay"
                      data-value={26}
                      onClick={[Function]}
                    >
                      26
                    </td>
                    <td
                      className="rdtDay"
                      data-value={27}
                      onClick={[Function]}
                    >
                      27
                    </td>
                    <td
                      className="rdtDay"
                      data-value={28}
                      onClick={[Function]}
                    >
                      28
                    </td>
                  </tr>
                  <tr>
                    <td
                      className="rdtDay"
                      data-value={29}
                      onClick={[Function]}
                    >
                      29
                    </td>
                    <td
                      className="rdtDay"
                      data-value={30}
                      onClick={[Function]}
                    >
                      30
                    </td>
                    <td
                      className="rdtDay rdtNew"
                      data-value={1}
                      onClick={[Function]}
                    >
                      1
                    </td>
                    <td
                      className="rdtDay rdtNew"
                      data-value={2}
                      onClick={[Function]}
                    >
                      2
                    </td>
                    <td
                      className="rdtDay rdtNew"
                      data-value={3}
                      onClick={[Function]}
                    >
                      3
                    </td>
                    <td
                      className="rdtDay rdtNew"
                      data-value={4}
                      onClick={[Function]}
                    >
                      4
                    </td>
                    <td
                      className="rdtDay rdtNew"
                      data-value={5}
                      onClick={[Function]}
                    >
                      5
                    </td>
                  </tr>
                </tbody>
                <tfoot>
                  <tr>
                    <td
                      className="rdtTimeToggle"
                      colSpan={7}
                      onClick={[Function]}
                    >
                      00:00:00
                    </td>
                  </tr>
                </tfoot>
              </table>
            </div>
          </div>
        </div>
      </div>
    </div>
    <button
      className="ui fluid primary button"
      onClick={[Function]}
      type="submit"
    >
      Filter
    </button>
  </form>
  <table
    className="ui celled collapsing sortable table"
    style={
      Object {
        "margin": "0",
        "marginLeft": "40px",
      }
    }
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <th
          className=""
          onClick={[Function]}
        >
          Log id
        </th>
        <th
          className=""
          onClick={[Function]}
        >
          Device id
        </th>
        <th
          className=""
          onClick={[Function]}
        >
          Adaption id
        </th>
        <th
          className=""
          onClick={[Function]}
        >
          Description
        </th>
        <th
          className=""
          onClick={[Function]}
        >
          Date
        </th>
      </tr>
    </thead>
    <tbody
      className=""
      data-cy="children"
    />
  </table>
</div>
`;
